packages/ui/src/composables/useColors.ts

```diff
@@ -144,8 +144,10 @@ export const useColors = () => {
     return getColorLightnessFromCache(color) > globalConfig.value.colors.threshold ? darkColor : lightColor
   }
 
-  const currentPresetName = computed(() => globalConfig.value.colors!.currentPresetName)
-
+  const currentPresetName = computed<string>({
+    get: () => globalConfig.value.colors!.currentPresetName,
+    set: (v:string) => { globalConfig.value.colors!.currentPresetName = v },
```

Comment:
I guess what we want to do is::
```suggestion
    set: (v: string) => { applyPreset(v) },
```